frontend/catalyst/from_plxpr/qreg_manager.py

```diff
@@ -188,3 +188,107 @@ def __setitem__(self, index: int, qubit: AbstractQbit):
     def __iter__(self):
         """Iterate over wires map dictionary"""
         return iter(self.wire_map.items())
+
+
+class QbitManager:
```

Comment:
Can we name this something new, more related to its scope and usage?
We have way too many qubit/qbit managers in the library already.

Reply:
Good idea @mlxd! This class manages `quantum.bit` rather than `quantum.register`. We already have `QRegManager` for registers, so this new class would handle q values. A few suggestions:
- QBitHandler
- QbitIRManager
- QbitValueManager
- QbitValueHandler

Reply:
Yes it's just to be analogous to the QregManager. Happy to change it but we should probably update both in that case.

Alternatively, there is the possibility that we merge the two classes and just allow the QregManager to have `None` for the qreg attribute.

Alternative names that describe the _function_ of these classes a bit better:
- QubitValueMap
- ValueSemanticConverter

Reply:
> we should probably update both in that case

Yeah, I believe there is actually a product effort focused on finding coherent names and definitions for different qubit managers and maps across the ecosystem. cc: @isaacdevlugt  

> QubitValueMap

I updated the name to this for now :+1:

Reply:
Did you look at merging the two classes @maliasadi?

Reply:
Done [55e2b6b](https://github.com/PennyLaneAI/catalyst/pull/1820/commits/55e2b6bb1f2eb29877e814e3219fc4fd69d408cd)